src/Console/Shell.php

```diff
@@ -640,6 +640,48 @@ public function err($message = null, $newlines = 1)
     }
 
     /**
+     * Convenience method for out() that wraps message between <info /> tag
+     *
+     * @param string|array|null $message A string or an array of strings to output
+     * @param int $newlines Number of newlines to append
+     * @param int $level The message's output level, see above.
+     * @return int|bool Returns the number of bytes returned from writing to stdout.
+     * @see http://book.cakephp.org/3.0/en/console-and-shells.html#Shell::out
+     */
+    public function info($message = null, $newlines = 1, $level = Shell::NORMAL)
+    {
+        return $this->out('<info>' . $message . '</info>', $newlines, $level);
+    }
+
+    /**
+     * Convenience method for out() that wraps message between <warning /> tag
+     *
+     * @param string|array|null $message A string or an array of strings to output
+     * @param int $newlines Number of newlines to append
+     * @param int $level The message's output level, see above.
+     * @return int|bool Returns the number of bytes returned from writing to stdout.
+     * @see http://book.cakephp.org/3.0/en/console-and-shells.html#Shell::out
+     */
+    public function warn($message = null, $newlines = 1, $level = Shell::NORMAL)
+    {
+        return $this->out('<warning>' . $message . '</warning>', $newlines, $level);
```

Comment:
I think a warning is often more a user warning type of thing and better be left in stdout.
It simply highlights in a color so you can more easily notice it.